### PR TITLE
Interface for selecting an app language

### PR DESCRIPTION
**Pull request by mrkvon**
Follow up of #921 

- [x] visitor can select a language in a header
  - the selected language will be saved in a cookie
- [x] user can select a language in account page
  - [x] the selected language will be saved in the database
  - [ ] the saved language will be loaded to i18n on signin or page reload #1054 
- [x] design of buttons (see [a proposed design](https://github.com/Trustroots/trustroots/pull/921#issuecomment-446142245))
  - [x] the language switch button in header needs positioning and some other details. i can't do that.

- API was updated in #962

### CODE REVIEW

**Comment by mrkvon** [comment]
Since we don't have a centralized place in React where we could load the user's language, this is as far as I can go now.

Added positioning. Ready for review and/or merge.

I disabled the display of dropdown-menu on hover for the language switch. Before it was disappearing because of a space between the button and the menu.

File: modules/core/client/components/LanguageSwitchPresentational.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      title={currentLanguage.name}
+      id={`dropdown-basic-${currentLanguage.code}`}
+      pullRight
+      className="disable-hover-display"
+    >
+      <Dropdown.Toggle className="btn-inverse">
+        {currentLanguage.name}
+      </Dropdown.Toggle>
+      <Dropdown.Menu>
+        {otherLanguages.map(language => (
```
Comment (simison) [inline modules/core/client/components/LanguageSwitchPresentational.js]: I think we could simplify this and list all the languages regardless, and just mark list item that matches `currentLanguage.code` with `active` -prop and it'll get highlighted.

It'll be double in toggle and in the list but it's okay, user can orient themself easier with list where items don't dissapear. (and code is simpler, yay!)

**Comment by simison** [comment]
Tried rebasing this but I don't think I did such a good job at it, so I'll leave it to ya. :-) 

This seems ready code-wise, but I'd be happy to give this visual inspection.

**Comment by mrkvon** [comment]
rebased, still some pending comments

File: modules/core/client/components/LanguageSwitchPresentational.js (hunk 1 above)
Comment (mrkvon) [reply modules/core/client/components/LanguageSwitchPresentational.js]: 247f92c88dce87906516c9a1521b2e624ab94d06

selector

**Comment by mrkvon** [comment]
Resolved.

The visual inspection would be nice.

Look into (top header for non-logged user) and to (user account). Both contain different-looking language switches (and the account switch additionally saves the user's locale to database (even though we don't use it, yet)).

**Pull request merged**